docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.9"
+
+services:
+  backend:
+    build: "https://github.com/maria-robobug/try-cb-python.git#DOC-8136"
+    depends_on:
+      db:
+        condition: service_healthy
+    ports:
+      - 8080:8080
+    environment:
+      - ARGS
+    command: $ARGS
```

Comment:
You will need to update `backend` and `db` definitions a bit, so we're in sync:

https://github.com/couchbaselabs/try-cb-python/pull/20/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3

Obviously ignore the `volume` part for the backend, as you won't need it in this project.